testsuite/tests/kuadrant/authorino/operator/tls/conftest.py

```diff
@@ -0,0 +1,119 @@
+"""Conftest for all TLS-enabled tests"""
+from urllib.parse import urlparse
+
+import pytest
+
+from testsuite.certificates import CFSSLClient, Certificate
+from testsuite.openshift.client import OpenShiftClient
+from testsuite.openshift.httpbin import TLSEnvoyHttpbin
+
+
+@pytest.fixture(scope="session")
+def create_secret(blame, request):
+    """Creates TLS secret from Certificate"""
+    def _create_secret(openshift: OpenShiftClient, certificate: Certificate, name):
```

Comment:
It was done to have an ability to work on multiple clusters but since that is nowhere near present/supported in the rest of the testsuite I will change it to a fixture.